tac/governing-documents/defining-adopters.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Defining Adopters
+parent: Governing Documents
+grand_parent: LF Decentralized Trust TAC
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Defining Adopters
+
+Adopters of an LF Decentralized Trust (LFDT) project are organizations or communities that successfully leverage the project as intended—whether integrating it into their systems, building on top of it, or offering it as a foundational component of a service or solution.
+
+To better understand how a project is adopted, we distinguish between **direct adopters** and **transitive adopters**. For example, if a service provider uses a decentralized identity project from LFDT as the backbone of a digital credentialing solution, the service provider is a *direct adopter*. If their customers use that solution without directly engaging with or deploying the underlying project, those customers are *transitive adopters*.
+
+* **Direct adopters** are responsible for actively developing, packaging, configuring, or deploying the LFDT project as part of their use.
+
+* **Transitive adopters** benefit from the project indirectly but are not responsible for its integration or operation. However, if a transitive adopter gains the ability to configure or extend the project due to the service provider’s implementation, they may become direct adopters as well.
+
+The intent of identifying adopters within the LFDT community is to gain insight into real-world, production-level usage of its projects or specifications. This helps assess project maturity, community engagement, and long-term viability within the decentralized trust ecosystem.
+
+In some cases, original sponsoring organizations or maintainers may also be recognized as adopters. The LFDT and the associated LF/DT projects’ technical and community leadership reserves the discretion to determine whether an adopter can offer meaningful feedback to guide the evolution of a project.
+
+Adopters of LFDT projects may include the following (noting that a single organization may fit more than one category):
+
+* **Ecosystem Participant** – An organization actively engaged in developing, deploying, or operating one or more LFDT projects.
+
+* **Service Provider** – An entity offering LFDT projects as part of a larger solution (e.g., a platform or product).
+
+* **Upstream or Downstream Project** – An open source project that depends on or integrates with an LFDT project for functionality, compatibility, or supply chain inclusion.
+
+* **End-user Organization** – A company or institution that uses an LFDT project to power part of its infrastructure, services, or internal workflows, without necessarily contributing upstream.
```

Comment:
To differentiate between this and `Service Provider` maybe add and `not as part of a platform or product offering`?

Reply:
and it was explained below with more detail?

Reply:
Thanks, Diane. I have attempted to capture this information in the document.